jwql/instrument_monitors/miri_monitors/cosmic_ray_monitor.py

```diff
@@ -0,0 +1,467 @@
+                                                        
+                                                            eff_time = jump_head['EFFEXPTM']
+                                                            
+                                                            # Get observation time info
+                                                            obs_start_time = jump_head['EXPSTART']
+                                                            obs_end_time = jump_head['EXPEND']
+                                                            start_time = julian.from_jd(obs_start_time, fmt='mjd')
+                                                            end_time = julian.from_jd(obs_end_time, fmt='mjd')
+                                                            
+                                                            cosmic_ray_num = len(jump_locs)
+                                                            #cosmic_ray_rate = get_cr_rate(jump_locs, eff_time)
+                                                            cosmic_ray_mags = get_cr_mags(jump_locs, jump_locs_pre, rate_data, jump_data, jump_head, nints)
+                                                            
+                                                            # Insert new data into database
+                                                            try:
+                                                                cosmic_ray_db_entry = {'entry_date': datetime.datetime.now(),
+                                                                    'aperture': aperture,
+                                                                        'source_file': file_name,
+                                                                            'obs_start_time': start_time,
+                                                                                'obs_end_time': end_time,
+                                                                                    'jump_count': cosmic_ray_num,
+                                                                                        'magnitude': cosmic_ray_mags
+                                                                                        }
+                                                                                            stats_table.__table__.insert().execute(cosmic_ray_db_entry)
+                                                                                            
+                                                                                                logging.info("Successfully inserted into database. \n")
+                                                                                                    except:
+                                                                                                        logging.info("Could not insert entry into database. \n")
```

Comment:
Is there a reason that everything is nested and has increasing indentation here?  It is a bit confusing to me as a reader.  Could provide some in-line comments to describe what is going on here?

Reply:
This is interesting. It does not look like this in my local copy. I wonder if it somehow got broken while pushing to github.... I'll look into this.